src/jobs/queue.go

```diff
@@ -339,3 +340,32 @@ func QueueBatchFinalizeIssueFlagging(batch *models.Batch, flagged []*models.Flag
 
 	return QueueSerial(jobs...)
 }
+
+// QueueCopyBatchForProduction sets the given batch to pending, then queues up
+// the necessary jobs to get it ready for a production load
+func QueueCopyBatchForProduction(batch *models.Batch, prodBatchRoot string) error {
+	var op = dbi.DB.Operation()
+	op.Dbg = dbi.Debug
+	op.BeginTransaction()
+	defer op.EndTransaction()
+
+	// We need batch status and staging purge flag to be updated instantly, not
+	// dependent on the speed of the job runner
+	batch.Status = models.BatchStatusPending
+	batch.NeedStagingPurge = true
+	var err = batch.SaveOp(op)
+	if err != nil {
+		return err
+	}
+
+	// Our sync job is special - it requires us to have exclusions, so we're just
+	// building a custom args list
+	var args = makeSrcDstArgs(batch.Location, filepath.Join(prodBatchRoot, batch.FullName()))
+	args[excludeArg] = `*.tif,*.tiff,*.TIF,*.TIFF,*.tar.bz,*.tar`
+
+	return QueueSerialOp(op,
+		PrepareBatchJobAdvanced(models.JobTypeValidateTagManifest, batch, nil),
+		PrepareJobAdvanced(models.JobTypeSyncDir, args),
+		PrepareBatchJobAdvanced(models.JobTypeSetBatchStatus, batch, makeBSArgs(models.BatchStatusPassedQC)),
```

Comment:
The issue for this mentioned setting a status other than passed_qc after sync, but looks like it's still going with that. Or am I misunderstanding the bullet in the issue? https://github.com/uoregon-libraries/newspaper-curation-app/issues/226

Reply:
You are correct! I decided I didn't actually care about renaming the status since its meaning is "close enough" to the same. I figured no sense burning time changing status names everywhere just to say "passed QC and then some".

Reply:
Haha, okay. That's fine. No need to split hairs.